docs/visual-testing/integrations/java.md

```diff
@@ -424,17 +428,177 @@ import com.saucelabs.visual.model.FullPageScreenshotConfig;
-        .withDelayAfterScrollMs(500)
-        .withDisableCSSAnimation(false)
-        .withHideAfterFirstScroll("#header")
-        .withHideScrollBars(false)
-        .withScrollLimit(5)
-        .build();
+    .withDelayAfterScrollMs(500)
+    .withDisableCSSAnimation(false)
+    .withHideAfterFirstScroll("#header")
+    .withHideScrollBars(false)
+    .withScrollLimit(5)
+    .build();
 options.enableFullPageScreenshots(config);
 visual.sauceVisualCheck("Long content page", options);
 ```
 
-<FullPageLimit />
+#### Mobile Native (beta)
+
+Methods available:
+- `withDelayAfterScrollMs(int delayAfterScrollMs)`: Delay in ms after scrolling and before taking screenshots. The default value is 0. We recommend using this option for lazy loading content.
+- `withNativeClipSelector(SelectorIn nativeClipSelector)`: Selector used to identify the first element to which clipping will be applied.
+- `withScrollElement(WebElement scrollElement)`: Scrollable element used for scrolling. The default is root element.
+- `withScrollLimit(int scrollLimit)`: Limit the number of screenshots taken for scrolling and stitching. The default value is 10. The value needs to be between 1 and 10.
+
+:::note
+It is recommended to define the `withScrollElement` as the appropriate scrollable container.
+:::
+
+<Tabs>
+    <TabItem value="ios" label="iOS">
+        ```java
+        import com.saucelabs.visual.CheckOptions;
+        import com.saucelabs.visual.model.FullPageScreenshotConfig;
+        
+        RemoteWebDriver driver;
+        ...
+        
+        WebElement scrollElement = driver.findElement(AppiumBy.xpath("//XCUIElementTypeCollectionView"));
+        CheckOptions options = new CheckOptions();
+        FullPageScreenshotConfig config = new FullPageScreenshotConfig.Builder()
+            .withScrollElement(scrollElement)
+            .withScrollLimit(5)
+            .build();
+        options.enableFullPageScreenshots(config);
+        visual.sauceVisualCheck("Long content page", options);
+        ```
+    </TabItem>
+    <TabItem value="android" label="Android">
+        ```java
+        import com.saucelabs.visual.CheckOptions;
+        import com.saucelabs.visual.model.FullPageScreenshotConfig;
+        
+        RemoteWebDriver driver;
+        ...
+        
+        WebElement scrollElement = driver.findElement(AppiumBy.xpath("//androidx.recyclerview.widget.RecyclerView")); 
+        CheckOptions options = new CheckOptions();
+        FullPageScreenshotConfig config = new FullPageScreenshotConfig.Builder()
+            .withScrollElement(scrollElement)
+            .withScrollLimit(5)
+            .build();
+        options.enableFullPageScreenshots(config);
+        visual.sauceVisualCheck("Long content page", options);
+        ```
+    </TabItem>
+</Tabs>
+
+Use only XPath selectors for ignore regions and clipping to an element.
+
+:::note
+On iOS, selectors must be contained within the `scrollElement`.
+:::
```

Comment:
I'd put this into a warning rather than a note. IMO it needs to be highlighted.

```suggestion
:::warning
On iOS, selectors **must** be contained within the `scrollElement`.
:::
```